Platform/TigerlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1575,11 +1585,11 @@ UpdateFspConfig (
 
   FspsConfig->SkipMpInit = 0;
 
-  FspsConfig->UsbTcPortEn = 0xf;
-  FspsConfig->ITbtPcieRootPortEn[0] = 0x1;
-  FspsConfig->ITbtPcieRootPortEn[1] = 0x1;
-  FspsConfig->ITbtPcieRootPortEn[2] = 0x1;
-  FspsConfig->ITbtPcieRootPortEn[3] = 0x1;
+  FspsConfig->UsbTcPortEn = 0x0;
+  FspsConfig->ITbtPcieRootPortEn[0] = 0x0;
+  FspsConfig->ITbtPcieRootPortEn[1] = 0x0;
+  FspsConfig->ITbtPcieRootPortEn[2] = 0x0;
+  FspsConfig->ITbtPcieRootPortEn[3] = 0x0;

```

Comment:
Remove these lines for UsbTcPortEn and ITbtPcieRootPortEn since they are same with FSP UPD default value, and user could change FSP to enable them if needed.

Reply:
Done removed, thanks.